waspc/data/Generator/templates/react-app/netlify.toml

```diff
@@ -0,0 +1,8 @@
+[build]
+  publish = "build/"
+
+[[redirects]]
```

Comment:
The thing is, since this is SPA, if somebody goes directly to url /login, by default it fails with 404 -> there is nothing there, there is a page only at `/index.html` therefore `/`. This redirect all requests to index.html. It is needed for any SPA that you host, and not just for Netlify, anywhere.